Python/thread_pthread.h

```diff
@@ -28,6 +30,9 @@
 #   include <lwp.h>             /* _lwp_self() */
 #elif defined(__DragonFly__)
 #   include <sys/lwp.h>         /* lwp_gettid() */
+#elif defined(__GNU__)
```

Comment:
Is `__GNU__` guaranteed to be defined only for GNU Hurd-based systems?

Reply:
Yes, it is the standard macro for testing GNU/Hurd